### DISCUSSION TITLE
Could not find PYTHON when build on windows

### DISCUSSION THREAD

**Discussion by Shurelol** [transmission/transmission#8508] [Support]
### What is the issue?

> 00:06:32.040   Performing configure step for 'psl'
00:06:32.118   Not searching for unused variables given on the command line.
00:06:32.165   -- Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.20348.
00:06:33.526   -- The C compiler identification is MSVC 19.29.30152.0
00:06:33.807   -- Detecting C compiler ABI info
00:06:35.732   -- Detecting C compiler ABI info - done
00:06:35.763   -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
00:06:35.763   -- Detecting C compile features
00:06:35.763   -- Detecting C compile features - done
00:06:36.216   CMake Error at CMakeLists.txt:30 (find_program):
00:06:36.216   -- Configuring incomplete, errors occurred!
00:06:36.216     Could not find PYTHON using the following names: python, python2, python3,
00:06:36.216     python3.7

but python3.12 already installed

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

4.0.4

**Comment by mikedld** [comment]
"Already installed" should mean that executing at least one of the listed commands (python, python2, python3, python3.7) in the same environment where you're trying to configure Transmission succeeds, i.e. that one of those binaries is in your PATH. Is that the case?
